src/converters/fromZigbee.ts

```diff
@@ -352,6 +396,7 @@ const converters1 = {
         convert: (model, msg, publish, options, meta) => {
             const payload: KeyValueAny = {};
             if (msg.data.hasOwnProperty('batteryPercentageRemaining') && (msg.data['batteryPercentageRemaining'] < 255)) {
```

Comment:
I'm thinking now, maybe we should always ignore the battery % reporting when `voltageToPercentage` is set since those will always conflict with each other.

Reply:
That vastly simplifies this PR, and avoids the need to have a new converter. I've force-pushed here as resolving merge conflicts wasn't worth the effort.